src/models/kafkaCluster.ts

```diff
@@ -12,7 +12,7 @@ export abstract class KafkaCluster extends Data implements IResourceBase {
   iconName: IconNames = IconNames.KAFKA_CLUSTER;
 
   abstract name: string;
-  abstract environmentId: string | undefined;
+  abstract environmentId: EnvironmentId | undefined;
```

Comment:
`KafkaCluster` model family now modeling `environmentId` as branded type `EnvironmentId`.

I ... do not know offhand why we support it as `... | undefined`, in that when would we ever not know a topic's environment? I wonder if this is holdover from when we weren't modeling local as an actual environment.

Reply:
Yeah, we shouldn't support `undefined` here anymore